### ISSUE TITLE
Alt text not accessable

### ISSUE THREAD

**Issue by Rimole** [mastodon/mastodon#35698]
### Steps to reproduce the problem

The alt-text is labelled with >>property="og:image:alt"<< instead of >>alt="..."<<.

### Expected behaviour

Alt text should be shown.

### Actual behaviour

Alt text is not shown.

### Detailed description

See https://forum.palemoon.org/viewtopic.php?f=70&t=32635

### Mastodon instance

any tested one

### Mastodon version

v4.4.3

### Browser name and version

Pale Moon 33.8.1.2 as well as Firefox 140.0.4 

### Operating system

Windows 11

### Technical details

See https://forum.palemoon.org/viewtopic.php?f=70&t=32635

**Comment by Gargron** [comment]
You are confusing the OpenGraph tag with the `<img />` tag. The `<img />` tag has `alt`.
